### PR TITLE
Use ReturnTypeExtension for mb_str_split

### PR DESCRIPTION
**Pull request by VincentLanglet**
Shouldn't `mb_str_split` work as `str_split` ? @ondrejmirtes 

I can add tests if you think it's right.

### CODE REVIEW

File: src/Type/Php/StrSplitFunctionReturnTypeExtension.php [hunk 1]
```diff
@@ -21,7 +21,8 @@ final class StrSplitFunctionReturnTypeExtension implements DynamicFunctionReturn
 
 	public function isFunctionSupported(FunctionReflection $functionReflection): bool
 	{
-		return $functionReflection->getName() === 'str_split';
+		return $functionReflection->getName() === 'str_split'
+			|| $functionReflection->getName() === 'mb_str_split';
```
Comment (ondrejmirtes) [inline src/Type/Php/StrSplitFunctionReturnTypeExtension.php:25]: Use in_array please.

**Review by ondrejmirtes — CHANGES_REQUESTED** [review]
And a test. See where str_split is already tested.

**Comment by VincentLanglet** [comment]
@ondrejmirtes The PR seems fine to me now. But all the test are failing weirdly.

**Comment by ondrejmirtes** [comment]
Thank you!

**Comment by ondrejmirtes** [comment]
I fixed the CI with: https://github.com/phpstan/phpstan-src/commit/ef6ca4abbc4db0a2f1ce2a2bdb45bd8ce8235378

Something weird is going on with latest version of nette/di...

**Comment by VincentLanglet** [comment]
> Thank you!

Thanks you too. 
I look forward the next release to fix all the false-positive I got when I change `str_plit` into `mb_str_split`.

**Pull request merged**